presto-docs/src/main/sphinx/develop/sidecar-protocol.rst

```diff
@@ -0,0 +1,19 @@
+==============================
+Presto Native Sidecar REST API
+==============================
+
+In a Presto native cluster, the coordinator sidecar process communicates with the
```

Comment:
Confused by the terminology of coordinator sidecar process. We have just a single co-ordinator process right ?